.github/workflows/build.yml

```diff
@@ -13,19 +13,27 @@ jobs:
     tests:
         runs-on: ubuntu-latest
 
-        name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}, Node ${{ matrix.node }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
         strategy:
             fail-fast: false
             matrix:
                 php: [7.4, 7.3]
+                symfony: [^4.4, ^5.2]
+                sylius: [~1.8.0, ~1.9]
                 node: [10.x]
                 mysql: [5.7, 8.0]
 
                 exclude:
                     -
                         php: 7.3
                         mysql: 8.0
+                    -
+                        sylius: ~1.8.0
+                        symfony: ^5.2
+                    -
+                        sylius: ~1.9
+                        php: 7.3
```

Comment:
Why did you add a conflict for Sylius 1.9 with PHP 7.3?

Reply:
Was green, but we droped php 7.3 in Sylius 1.9

Reply:
Nope, we will drop PHP 7.3 in next minor version, we still support this version in 1.9: https://github.com/Sylius/Sylius/blob/1.9/composer.json#L22